modules/monitoring/manifests/checks.pp

```diff
@@ -109,6 +109,52 @@
   }
   # END signon
 
+  # START search
+
+  # On average 326 new documents were created per day in 2017, in total.
+  # The govuk index will only receive a fraction of these until we start
+  # indexing whitehall content in it.
+  icinga::check::graphite { 'check_rummager_govuk_index_size_changed':
+    target              => 'absolute(diffSeries(stats.gauges.govuk.app.rummager.govuk_index.docs.count, timeShift(stats.gauges.govuk.app.rummager.govuk_index.docs.count, "3d"))',
+    warning             => 1500,
+    critical            => 15000,
```

Comment:
what do these values mean, are the change in the number of documents? or percentage? also, the critical warning feels too big compared to the warning.

Reply:
These are changes in the number of documents.